### ISSUE TITLE
investigate TestSecurity fail

### ISSUE THREAD

**Issue by srl295** [unicode-org/unicodetools#193]
https://github.com/unicode-org/unicodetools/pull/174#issuecomment-1009365758 found that #151 may have broke CI, _at least with an updated CLDR_

**Comment by srl295** [comment]
Got a little further, but still https://github.com/unicode-org/unicodetools/runs/5101384609?check_suite_focus=true#step:8:517 

```
Caused by: com.ibm.icu.util.ICUUncheckedIOException: java.io.FileNotFoundException: confusables.txt/home/runner/work/unicodetools/unicodetools/unicodetools/data/security/dev (No such file or directory)
[518](https://github.com/unicode-org/unicodetools/runs/5101384609?check_suite_focus=true#step:8:518)
```

**Comment by markusicu** [comment]
That looks like the code is doing filename + separator + path rather than the other way around.

When it's the right way around, it should read https://github.com/unicode-org/unicodetools/blob/main/unicodetools/data/security/dev/confusables.txt

**Comment by srl295** [comment]
> That looks like the code is doing filename + separator + path rather than the other way around.
> 
> When it's the right way around, it should read https://github.com/unicode-org/unicodetools/blob/main/unicodetools/data/security/dev/confusables.txt

Just the conclusion I came to.  https://github.com/unicode-org/cldr/pull/1745

There was another issue where the CLDR invoker needed to include 'seed' otherwise `ks-Deva` could not be found (that was the original error).   I fix this in #174

**Comment by srl295** [comment]
fixed in #174
